tests/plando/boss_rush_dc_mq.json

```diff
@@ -4,7 +4,8 @@
   "shuffle_dungeon_entrances":               true,
   "open_forest":                             "open",
   "mq_dungeons":                             1,
-  "dungeon_shortcuts":                       ["dodongos_cavern"],
+  "dungeon_shortcuts":                       "choice",
+  "dungeon_shortcuts_list":                       ["dodongos_cavern"],
```

Comment:
These all should also be aligned.

Reply:
Fix : https://github.com/TestRunnerSRL/OoT-Randomizer/pull/1493/commits/b3958c886644d5d247a9adc47cb33b4e9a50bafb